src/js/messaging/containers/Thread.jsx

```diff
@@ -277,18 +268,18 @@ export class Thread extends React.Component {
   }
 
   render() {
-    if (this.props.isNewMessage && this.props.loadingRecipients) {
-      return <LoadingIndicator message="Loading the application..."/>;
-    }
-
     const loading = this.props.loading;
```

Comment:
Do you have preferences for using destructuring assignment up front for all of the first line variables coming from `this.props`, e.g. `isNewMessage, loading, message`, etc. instead of having to reference `this.props` everywhere below?